config/defaults

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+export DATA_ROOT=${DATA_ROOT:-$ROOTDIR/data}
```

Comment:
There is a version of this at `sidecar:/l1/govdocs/zcode/config/.env`. It includes some things that aren't being set here because they aren't being used anywhere.